### PR TITLE
Added an "--output" param to specify destination file of prepare command

### PR DESCRIPTION
**Pull request by tlorreyte**
Hi, web/data.json was hardcoded in prepare command, i just added another not required parameter to specify the destination json file.

### CODE REVIEW

File: commands/prepare.py [hunk 1]
```diff
@@ -667,13 +667,21 @@ def prepare(account, config, outputfilter):
...
         json.dump(cytoscape_json, outfile, indent=4)
 
 
 def run(arguments):
     # Parse arguments
     parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--output", 
+        help="Output data file path",
+        required=False,
```
Comment (KevinHock) [inline commands/prepare.py:680]: All `--` args are not required, so for consistency with other `add_argument` calls, maybe we should remove this line.

**Review by KevinHock — APPROVED** [review]

**Comment by tlorreyte001** [comment]
Hi @KevinHock, thank you for reviewing, i deleted the required line.
Have a nice day !

**Comment by tlorreyte001** [comment]
Hi @KevinHock, is it ok for you dude ?

**Comment by w0rmr1d3r** [comment]
Hello @tlorreyte001 !
Careful, exists this issue https://github.com/duo-labs/cloudmapper/issues/802 with this PR https://github.com/duo-labs/cloudmapper/pull/854 saying that the functionality you are modifying in this PR, is no longer maintained.
Just so you know 😃 

Cheers!

**Comment by 0xdabbad00** [comment]
I'm closing this as this functionality is no longer maintained.

**Pull request closed without merging**